doc/6/getting-started/.react/cypress/fixtures/Sauron.json

```diff
@@ -0,0 +1,5 @@
+{
+    "body": {
+        "text": "I am no man"
```

Comment:
Did sauron ever say that ? :trollface:

Reply:
Here sauron is the receiver of this message :p 
It's juste that instead of the vuejs getting started, this one doesn't implement the login ! So I removed the username of the receiver in the fixtures but I didn't change the filename to keep the same as the VueJS getting started :)